backend/src/middleware/categories.ts

```diff
@@ -1,9 +1,19 @@
-/* eslint-disable @typescript-eslint/no-unsafe-return */
-/* eslint-disable @typescript-eslint/no-unsafe-call */
-import CONFIG from '@src/config'
+import type { Context } from '@src/server'
 
-const checkCategoriesActive = (resolve, root, args, context, resolveInfo) => {
-  if (CONFIG.CATEGORIES_ACTIVE) {
+type Resolver = (
```

Comment:
this must be defined somewhere - we should not define it ourselfs

Reply:
There are ways to correctly set types for GraphQL resolvers based on your schema. But I found it a bit out of scope for this PR.

Reply:
leave it at `any`? You can ignore it in the file and as soon as we properly type our apollo properly it shows up as no longer needed?!
It seems forced.